crates/bitwarden-uniffi/src/platform/fido2.rs

```diff
@@ -0,0 +1,246 @@
+use std::sync::Arc;
+
+use bitwarden::{
+    error::Result as BitResult,
+    platform::fido2::{
+        CheckUserOptions, ClientData, GetAssertionRequest, GetAssertionResult,
+        MakeCredentialRequest, MakeCredentialResult,
+        PublicKeyCredentialAuthenticatorAssertionResponse,
+        PublicKeyCredentialAuthenticatorAttestationResponse,
+    },
+    vault::{Cipher, CipherView, Fido2Credential, Fido2CredentialView},
+};
+
+use crate::{error::Result, Client};
+
+#[derive(uniffi::Object)]
+pub struct ClientFido2(pub(crate) Arc<Client>);
```

Comment:
Let's document that everything in this struct is stubbed and not actually working.

Reply:
Done